src/App/pages/ReposSearchPage/ReposSearchPage.css

```
@@ -0,0 +1,46 @@
+.search-form{
+    margin: 20px auto;    
+}
+    .search-form__content{
```

для удобство можно стили внутри стилей определять таким образом: 

```suggestion
    &__content{
```